src/backend/distributed/executor/executor.c

```diff
@@ -401,14 +402,6 @@ CitusExecScan(CustomScanState *node)
 		{
 			targetPoolSize = 1;
 		}
-		else if (ForceMaxQueryParallelization)
```

Comment:
comment is not targeting this line, but more generic: Should we replace all `ForceMaxQueryParallelization` with  `UseSeparateConnectionPerPlacement` . We have 1-2 extra uses in the `unified_executor` branch